### PR TITLE
133 error messages user friendly and targeted

### PR DESCRIPTION
**Pull request by sensei100**

### CODE REVIEW

File: app/controllers/errors_controller.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class ErrorsController < ApplicationController
+  def not_found
+    render(:status => 404)
+  end
+
+  def internal_erver_error
```
Comment (bibliotechy) [inline app/controllers/errors_controller.rb]: @sensei100 looks like an "s" was missed both here and in the view file name

File: config/alma.yml.example [hunk 2]
```diff
@@ -1,5 +1,5 @@
 default: &default
-  apikey: "EX_LIBRIS_DEVELOPERS_NETWORK_APIKEY"
+  apikey: "l7xxb45c21271ab94ddfb7fcb7221e5b2ef6"
```
Comment (bibliotechy) [inline config/alma.yml.example]: OK, this api key should not be committed in the example. It should only have been changed in the `alma.yml` you copied this file to, not in the example.

**Comment by sensei100** [comment]
Those things should be fixed now.

File: config/routes.rb [hunk 3]
```diff
@@ -1,5 +1,9 @@
 Rails.application.routes.draw do
 
+  get 'errors/not_found'
+
+  get 'errors/internal_erver_error'
```
Comment (bibliotechy) [inline config/routes.rb]: I don't think we need these get routes in addition to the `match '/404'` and `match '/500'` below.

**Comment by sensei100** [comment]
I removed the get routes for errors.

**Pull request merged**